views/js/controller/creator/views/testpart.js

```diff
@@ -115,7 +115,8 @@ function($, _, defaults, actions, sectionView, templates, qtiTestHelper){
                         identifier : qtiTestHelper.getAvailableIdentifier(modelOverseer.getModel(), 'assessmentSection', defaults().sectionIdPrefix),
                         title : `${defaults().sectionTitlePrefix} ${sectionIndex + 1}`,
                         index : 0,
-                        sectionParts : []
+                        sectionParts : [],
+                        visible: true
```

Comment:
do we need also a manifest update?

Reply:
As I understand, now we no need it. From `New dependencies management and versioning` `Versions in the manifest file are no longer counted and will be removed soon.`

Reply:
No longer needed.